conf/nginx-webmail.conf

```diff
@@ -0,0 +1,21 @@
+	# Roundcube Webmail configuration.
```

Comment:
The contents of this file (conf/nginx-webmail.conf) seem to be duplicated in conf/nginx-alldomains.conf.

It would be better if there was only one source of this code for easier management (we don't have to remember to edit things in two places at once).

Reply:
Yeah, although it's slightly different, as here roundcube is being served from / instead of /mail.

Maybe I can find something better for this though..

Reply:
This is the reason why I'm not merging the PR. If there's a way to do it that doesn't duplicate any Roundcube config, then I'd be open to this.

Reply:
Okay, it looks like it will be hard to have this not duplicated, therefore I have come up with three alternatives, altough I am not sure on how to proceed. Maybe you can give an opinion about what you would like best:

1. Serve webmail on webmail.example.com/mail, and have it redirect there from webmail.example.com
2. Only serve webmail on webmail.example.com and have a redirect to there from example.com/mail